scripts/_common.sh

```
@@ -0,0 +1,686 @@
+
+  info "apt update"
+  sudo apt update || true
+  APT_HAS_NEW_REPOSITORY=0
+
+  info "apt upgrade"
+  sudo apt upgrade -y
+}
+
+# requires sudo
+function apt_install() {
+  for pkg in $*; do
```

Shellcheck (0.10.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)